### PR TITLE
[14.0][ADD] product_supplierinfo_for_customer_group_fix

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
If you install product_supplierinfo_for_customer and then product_supplierinfo_group
"Customer Pricelist" model behaviour will be broken, because product_supplierinfo_group
changes product.supplierinfo model fields definitions, making it related to
"Supplierinfo group".

This module restores default product.supplierinfo fields definitions and makes
product_supplierinfo_group compatible with product_supplierinfo_for_customer.

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
functional ok!

solves https://github.com/OCA/product-attribute/issues/1135

@OCA/product-maintainers can you provide a feedback on the module name? 

it's a module ensuring compatibility of product_supplierinfo_for_customer with product_supplierinfo_group, without implementing the functionalities of the latter in the former

@pedrobaeza what do you think? thanks!

**Review by elvise — APPROVED** [review]
LGTM!

**Comment by francesco-ooops** [comment]
@ilyasProgrammer can you set to open?

@mourad-ehm @kevinkhao @sebastienbeau can you provide an opinion? Thank you!

**Comment by francesco-ooops** [comment]
@rousseldenis could you provide a feedback on this one too? thanks!

**Comment by francesco-ooops** [comment]
@pedrobaeza any hint on module naming? it's to preserve a Tecnativa module to keep working as intended :)

**Comment by pedrobaeza** [comment]
Name without ending `_fix` sounds good.

**Comment by ilyasProgrammer** [comment]
updated

**Comment by francesco-ooops** [comment]
@OCA/product-maintainers can we merge? thanks!

**Comment by francesco-ooops** [comment]
@rousseldenis good for merge?

File: product_supplierinfo_for_customer_group/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "and product_supplierinfo_group",
+    "version": "14.0.1.0.0",
+    "author": "Ilyas, Ooops404, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "category": "Sales Management",
+    "license": "AGPL-3",
+    "depends": ["product_supplierinfo_for_customer", "product_supplierinfo_group"],
+    "data": [],
+    "installable": True,
+    "autoinstall": True,
```
Comment (rousseldenis) [inline product_supplierinfo_for_customer_group/__manifest__.py]:
```suggestion
    "auto_install": True,
```

File: product_supplierinfo_for_customer_group/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "Supplierinfo Group compatibility",
+    "summary": "Fixes compatibility of product_supplierinfo_for_customer "
+    "and product_supplierinfo_group",
+    "version": "14.0.1.0.0",
+    "author": "Ilyas, Ooops404, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/product-attribute",
+    "category": "Sales Management",
+    "license": "AGPL-3",
+    "depends": ["product_supplierinfo_for_customer", "product_supplierinfo_group"],
+    "data": [],
```
Comment (rousseldenis) [inline product_supplierinfo_for_customer_group/__manifest__.py]: Can remove void entries

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review.

I would have added tests to ensure the behaviour is good.

**Comment by ilyasProgrammer** [comment]
updated

**Comment by francesco-ooops** [comment]
@ilyasProgrammer please squash commits

**Comment by francesco-ooops** [comment]
@rousseldenis good for you?

**Comment by francesco-ooops** [comment]
@rousseldenis should be good for merge now :)

**Comment by francesco-ooops** [comment]
@rousseldenis gentle reminder

**Review by rousseldenis — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@rousseldenis can you retry? :)

**Comment by francesco-ooops** [comment]
@OCA/product-maintainers merge?

**Comment by francesco-ooops** [comment]
@rousseldenis any idea what's wrong with this?

**Comment by francesco-ooops** [comment]
@pedrobaeza can you take a look at failed merge attempts? thanks

**Comment by pedrobaeza** [comment]
Please rebase to see in this PR what is happening.

**Comment by francesco-ooops** [comment]
@rousseldenis can you launch merge again please?

**Comment by pedrobaeza** [comment]
I think here it's happening the same problem as in https://github.com/OCA/knowledge/pull/381, with a continuous flooding of POT update. cc @sbidoul

**Comment by pedrobaeza** [comment]
I think I know what is happening: `product_supplierinfo_for_customer` is in the rebel module group, and your module is inheriting on it, so it's being installed in both environments, and thus, the POT is update in one build and the other with different things. I don't know why it should be put in the rebel part. You can try to remove it. The other option is to put your new modules that depend on it also on the rebel group, but try first to remove the rebel hack.

**Comment by ilyasProgrammer** [comment]
@pedrobaeza sorry but what does it mean: "remove the rebel hack" ?

**Comment by pedrobaeza** [comment]
Run `copier update` and remove the module in the rebel module group when asked.

**Comment by ilyasProgrammer** [comment]
updated

**Pull request merged**